dashboard/app/models/script.rb

```diff
@@ -1361,7 +1367,13 @@ def self.clear_cache
   end
 
   def localized_title
```

Comment:
this method isn't really "just" localizing any more, so I could call this method something else like `title_for_display` or `localized_title_for_display` if that's better. any input @Hamms (in addition to assigned reviewers) on naming? it seems like we'll have more methods like this which both localize and do additional processing for display once we are adding the markdown preprocessor layer for content moved over from CB like doclinks, resoure links, vocab links.

Reply:
Without much background on this, `title_for_display` sounds good to me, it seems to convey that it does whatever is needed to get the title properly "rendered".

I'm also wondering if there are other places in the code that use this method that wouldn't want the numbering, for some reason. If so it might make sense to keep the old `localized_title` as is, and make a new method `title_for_display` which calls it and does the additional prefix stuff.

Reply:
the reason I think that we will want this everywhere is that today the unit names already contain prefixes like "CSP Unit 1 - ". this flag is not going to be enabled retroactively for any unit groups, and will be enabled by default for new unit groups, so it seems like the current implementation will give the most similar behavior to what we had earlier. the only exception I saw was for the edit page, where we definitely want the raw unit name. I'll do another check for any other possible problem areas, but I guess I am saying that I think by default it makes the most sense to keep the unit prefix anywhere user-visible since that most closely matches the existing style of CSD / CSP units where the prefix has historically been visible as part of the raw unit name.